src/components/utilities.js

```diff
@@ -9,7 +9,11 @@ export function ResultWrapper(props) {
 
   if (field.type === 'text' && field.value !== undefined) {
     // replace "&", "|", "(", ")", "*", "'", """ and duplicated whitespace with only one whitespace
-    const values = field.value.replaceAll(/&|\*|\(|\)|\||"|'/g, "").replaceAll(/\s+/g, " ").split(" ");
+    let fieldValue = field.value;
+    if (field.field === "id" && value !== undefined){
+      fieldValue = field.value.replace("\\","")
```

Comment:
If we are rendering the identifier result, since the `field.value` is storing the escaped identifier (which we sent to the server), remove the backslash

Reply:
Is this still required now that we're using the returned value from the server instead of the one we sent to the server?